tap-snapshots/test/v8-to-istanbul.js.test.cjs

```diff
@@ -279,8 +279,8 @@ Object {
       "line": 35,
       "loc": Object {
         "end": Object {
-          "column": 0,
-          "line": 37,
+          "column": 29,
```

Comment:
The start of this branch is left unchanged, the end is slightly tighter.
![image](https://user-images.githubusercontent.com/160981/146623466-36ad49c6-75b0-4ec5-bcfb-8d8cf2b9e660.png)
I think that's a side effect of trimming the v8 range. Overall the range still seems OK to me
...maybe even better not being greedy?